pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/template/base/pensjonlatex/Settings.kt

```diff
@@ -8,24 +8,32 @@ import no.nav.pensjon.brev.template.Language
 import no.nav.pensjon.brev.template.dsl.*
 import no.nav.pensjon.brev.template.dsl.expression.*
 
+object Setting {
```

Comment:
Kan vel legge inn settings for resten i samme slengen? Eller var det en spesiell grunn til at Sakspart ble valgt?

Reply:
Det er i forbindelse med HTML-rendering. Trenger disse egenskapene i rendering av HTML, legger til flere om de trengs.